### PR TITLE
553851/add api check

### PR DESCRIPTION
**Pull request by ruspl-afed**
This is an attempt to setup automated API checks 
1) empty src/ folder added to a number of bundles to workaround tycho failure - there may be more elegant way to configure this, we may want to investigate it later
2) api* profiles were added to the parent pom - still under configuration

### CODE REVIEW

File: bundles/org.eclipse.passage.lbc.server/.classpath [hunk 1]
```diff
@@ -1,17 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.lbc.server/.classpath]: Removed by JDT

**Comment by eparovyshnaya** [comment]
There are lots of files changed in at leastt two different ways. Can I kindly ask you to describe chages in general?

File: bundles/org.eclipse.passage.lic.api/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Automatic-Module-Name: org.eclipse.passage.lic.api
 Bundle-ManifestVersion: 2
 Bundle-SymbolicName: org.eclipse.passage.lic.api
-Bundle-Version: 0.6.0.qualifier
```
Comment (eparovyshnaya) [inline bundles/org.eclipse.passage.lic.api/META-INF/MANIFEST.MF]: Just a bit confused - why the patch version has been bumped?

Comment (ruspl-afed) [reply bundles/org.eclipse.passage.lic.api/META-INF/MANIFEST.MF]: You are right. I still have problems with checker configuration, to be fixed

**Comment by ruspl-afed** [comment]
> @ruspl-afed There are lots of files changed in at least in two different ways. Can I kindly ask you to describe chages in general?

@eparovyshnaya thanks for the friendly reminder! Added description to the PR, currently it is in the WIP state

**Comment by ruspl-afed** [comment]
@eparovyshnaya , I suggest to merge this one is it provides version-check for the most important components: LIC and LDC
There are a lot of things to do (to be submitted in bugzilla):
1) provide composite p2 repository for all passage components
2) ensure pack200 is enabled for passage bundles and features
3) switch builds to the platform-like notation I,S,R with timestamp-based suffix
4) employ version-check for all the passage bundles and features
5) employ api-check for all the passage bundles and features

**Comment by eparovyshnaya** [comment]
Closes [553851](https://bugs.eclipse.org/bugs/show_bug.cgi?id=553851)

**Pull request merged**